### ISSUE TITLE
Generate popular OSS project's CRDs

### ISSUE THREAD

**Issue by shinebayar-g** [konfjs/k8skonf#7]

**Comment by rassie** [comment]
I'm torn on this. I guess I know why the wish is there, but since I've followed the evolution of https://github.com/jsonnet-libs for Jsonnet, which has pretty much always been way behind the times, I'd be wary of the effort involved in keeping the whole bunch up-to-date.

In general, I find `cdk8s`-style importing way superior, I feel more in control of my dependencies and I can always re-export them if I need them elsewhere. I've only had three problems with the approach in `cdk8s`, all of which are solvable: downloading Helm chart every single synth, depending on a basically dead https://github.com/crdsdev/doc and not supporting Git repositories where CRDs are only provided as Kustomize directories.

**Comment by shinebayar-g** [comment]
Yeah cdk8s style declarative import is already [there](https://github.com/konfjs/k8skonf/blob/86103756a3f26277957ed54021d7f3d225ddb8a2/examples/k8skonfig.yaml) btw.

As you said, prebuilt packages are meant for developing libraries/components.
